packages/server/package.json

```diff
@@ -47,7 +47,7 @@
     "@basemaps/landing": "^6.28.1"
   },
   "dependencies": {
-    "sharp": "^0.30.2"
+    "sharp": "^0.30.6"
```

Comment:
No "@types/sharp" required here?

Reply:
its not actually used here, but its used as a way of forcing the dependency to be installed.

We bundle the code a single js file. which removes all the deps from it